src/verify.rs

```diff
@@ -70,5 +70,6 @@ mod execute_block;
 
 pub mod aura;
 pub mod babe;
+pub mod babe_current_epoch;
```

Comment:
This module should belong to `src/chain/chain_information`.
Could you please rename the existing `chain_information::babe` module to `babe_config` (and same from `aura` to `aura_config` for consistency), and put it there instead?